### ISSUE TITLE
c3po mortality modeling sprint

### ISSUE THREAD

**Issue by erikr** [aguirre-lab/ml4c3#1034]
Train on all MGB data.

## 1y mortality
ConvNet
- [x] Training
- [x] Done
 
ResNet
- [x] Training
- [x] Done

DenseNet
- [x] Training
- [x] Done

ECGNet
- [x] Training
- [x] Done

## 5y mortality
ConvNet
- [x] Training
- [x] Done
 
ResNet
- [x] Training
- [x] Done

DenseNet
- [x] Training
- [x] Done

ECGNet-SOTA
- [x] Training
- [x] Done

**Comment by erikr** [comment]
We checked off `MGH+BWH 1y mortality: ResNet` and `MGH+BWH 1y mortality: ECGNet-SOTA`, but I don't see them in Dropbox.

Could you move the results to ~/dropbox/c3po`? Thanks.

After the current model finishes training on Anduril, I will consolidate the results folders into `~/dropbox/c3po/results`.

**Comment by StevenSong** [comment]
I think just a glitch or typo, I never ran any c3po models

**Comment by erikr** [comment]
1yr training on anduril

when de-id ECGs finish xfering to Quanta, will start more models there

**Comment by erikr** [comment]
ConvNet finished:
- took 6.4 days to complete 119 epochs
- training AUC = 0.838
- test AUC = 0.832.

I am not sure why the validation loss is lower than the training loss:

![Screen Shot 2021-01-07 at 6 35 43 AM](https://user-images.githubusercontent.com/1253422/103888344-93436380-50b2-11eb-8988-d426eabf2485.png)
